For the second command
```
bazel test //app:TopicLessonsFragmentTest --android_databinding_use_v3_4_args --experimental_android_databinding_v2 --java_header_compilation=false --noincremental_dexing --define=android_standalone_dexing_tool=d8_compat_dx
```
The output is

```
DEBUG: Rule 'io_bazel_stardoc' indicated that a canonical reproducible form can be obtained by modifying arguments commit = "4378e9b6bb2831de7143580594782f538f461180", shallow_since = "1570829166 -0400" and dropping ["tag"]
DEBUG: Repository io_bazel_stardoc instantiated at:
  E:/android/oppia-android-two/WORKSPACE:45:35: in <toplevel>
  C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/io_bazel_rules_kotlin/kotlin/internal/repositories/download.bzl:115:10: in kt_download_local_dev_dependencies
  C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/bazel_tools/tools/build_defs/repo/utils.bzl:201:18: in maybe
Repository rule git_repository defined at:
  C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/bazel_tools/tools/build_defs/repo/git.bzl:199:33: in <toplevel>
DEBUG: Rule 'robolectric' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "a7eac9418cc0b6d2527699986d5937efcebae927c3917c8bd3cb7663ee65ab63"
DEBUG: Repository robolectric instantiated at:
  E:/android/oppia-android-two/WORKSPACE:91:13: in <toplevel>
Repository rule http_archive defined at:
  C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
DEBUG: Rule 'tools_android' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1594238320 -0400"
DEBUG: Repository tools_android instantiated at:
  E:/android/oppia-android-two/WORKSPACE:102:15: in <toplevel>
Repository rule git_repository defined at:
  C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/bazel_tools/tools/build_defs/repo/git.bzl:199:33: in <toplevel>
INFO: Repository maven instantiated at:
  E:/android/oppia-android-two/WORKSPACE:125:14: in <toplevel>
  C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/rules_jvm_external/defs.bzl:89:19: in maven_install
Repository rule coursier_fetch defined at:
  C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/rules_jvm_external/coursier.bzl:620:33: in <toplevel>
INFO: Repository 'maven' used the following cache hits instead of downloading the corresponding file.
 * Hash '8f35f92fb8e021f96b3aa8145c66c3b2e29295baabb28ff50569e613438afcbd' for https://github.com/coursier/coursier/releases/download/v2.0.0-RC3-4/coursier.jar
If the definition of 'maven' was updated, verify that the hashes were also updated.
ERROR: An error occurred during the fetch of repository 'maven':
   Traceback (most recent call last):
        File "C:/users/syed farees hussain/_bazel_syed farees hussain/zcnrcatv/external/rules_jvm_external/coursier.bzl", line 364, column 13, in _coursier_fetch_impl
                fail("Error while fetching artifact with coursier: " + exec_result.stderr)
Error in fail: Error while fetching artifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_annotation_annotation in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifact w
ith coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_databinding_databinding_adapters in repository @maven which failed to fetch. no such package '@maven//': Error while fetching
 artifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_databinding_databinding_common in repository @maven which failed to fetch. no such package '@maven//': Error while fetching a
rtifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_databinding_databinding_compiler in repository @maven which failed to fetch. no such package '@maven//': Error while fetching
 artifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_databinding_databinding_runtime in repository @maven which failed to fetch. no such package '@maven//': Error while fetching
artifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_navigation_navigation_fragment in repository @maven which failed to fetch. no such package '@maven//': Error while fetching a
rtifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_navigation_navigation_ui in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifac
t with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_test_espresso_espresso_contrib in repository @maven which failed to fetch. no such package '@maven//': Error while fetching a
rtifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_test_espresso_espresso_core in repository @maven which failed to fetch. no such package '@maven//': Error while fetching arti
fact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_test_espresso_espresso_intents in repository @maven which failed to fetch. no such package '@maven//': Error while fetching a
rtifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_test_ext_junit in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifact with cou
rsier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:androidx_test_runner in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifact with coursi
er: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:com_github_bumptech_glide_mocks in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifact
with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:com_google_truth_truth in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifact with cour
sier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:org_jetbrains_kotlin_kotlin_test_junit in repository @maven which failed to fetch. no such package '@maven//': Error while fetching ar
tifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:org_jetbrains_kotlin_kotlin_reflect in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artif
act with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:org_jetbrains_kotlinx_kotlinx_coroutines_test in repository @maven which failed to fetch. no such package '@maven//': Error while fetc
hing artifact with coursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:org_mockito_mockito_core in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifact with co
ursier: Timed out
ERROR: E:/android/oppia-android-two/app/BUILD.bazel:1092:9: //app:TopicLessonsFragmentTest depends on @maven//:org_robolectric_annotations in repository @maven which failed to fetch. no such package '@maven//': Error while fetching artifact with
 coursier: Timed out
ERROR: Analysis of target '//app:TopicLessonsFragmentTest' failed; build aborted: Analysis failed
INFO: Elapsed time: 602.467s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)

```

--- a/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt
+++ b/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt
@@ -166,10 +166,10 @@ class TopicLessonsFragmentTest {
   }
 
   @Test
+  @Config(qualifiers = "land-xxhdpi")
   fun testLessonsPlayFragment_loadRatiosTopic_configurationChange_storyName_isCorrect() {
     launch<TopicActivity>(createTopicActivityIntent(internalProfileId, RATIOS_TOPIC_ID)).use {
       clickLessonTab()
-      onView(isRoot()).perform(orientationLandscape())
       verifyTextOnStorySummaryListItemAtPosition(itemPosition = 1, stringToMatch = "Ratios: Part 1")
     }
   }
@@ -306,12 +306,12 @@ class TopicLessonsFragmentTest {
   }
 
   @Test
+  @Config(qualifiers = "land-xxhdpi")
   fun testLessonsPlayFragment_loadRatiosTopic_clickExpandListIconIndex1_configurationChange_chapterListIsVisible() { // ktlint-disable max-line-length
     launch<TopicActivity>(createTopicActivityIntent(internalProfileId, RATIOS_TOPIC_ID)).use {
       clickLessonTab()
       clickStoryItem(position = 1, targetViewId = R.id.chapter_list_drop_down_icon)
       scrollToPosition(position = 1)
-      onView(isRoot()).perform(orientationLandscape())
       onView(
         atPositionOnView(
           R.id.story_summary_recycler_view,

--- a/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt
+++ b/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt
@@ -20,6 +20,7 @@ import androidx.test.espresso.intent.matcher.IntentMatchers.hasExtra
 import androidx.test.espresso.matcher.ViewMatchers.hasDescendant
 import androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA
 import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.ext.junit.runners.AndroidJUnit4
@@ -49,6 +50,7 @@ import org.oppia.android.app.story.StoryActivity
 import org.oppia.android.app.topic.TopicActivity
 import org.oppia.android.app.topic.TopicTab
 import org.oppia.android.app.utility.EspressoTestsMatchers.withDrawable
+import org.oppia.android.app.utility.OrientationChangeAction.Companion.orientationLandscape
 import org.oppia.android.domain.classify.InteractionsModule
 import org.oppia.android.domain.classify.rules.continueinteraction.ContinueModule
 import org.oppia.android.domain.classify.rules.dragAndDropSortInput.DragDropSortInputModule
@@ -168,6 +170,7 @@ class TopicLessonsFragmentTest {
   fun testLessonsPlayFragment_loadRatiosTopic_configurationChange_storyName_isCorrect() {
     launch<TopicActivity>(createTopicActivityIntent(internalProfileId, RATIOS_TOPIC_ID)).use {
       clickLessonTab()
+      onView(isRoot()).perform(orientationLandscape())
       verifyTextOnStorySummaryListItemAtPosition(itemPosition = 1, stringToMatch = "Ratios: Part 1")
     }
   }
@@ -308,6 +311,7 @@ class TopicLessonsFragmentTest {
   fun testLessonsPlayFragment_loadRatiosTopic_clickExpandListIconIndex1_configurationChange_chapterListIsVisible() { // ktlint-disable max-line-length
     launch<TopicActivity>(createTopicActivityIntent(internalProfileId, RATIOS_TOPIC_ID)).use {
       clickLessonTab()
+      onView(isRoot()).perform(orientationLandscape())
       clickStoryItem(position = 1, targetViewId = R.id.chapter_list_drop_down_icon)
       scrollToPosition(position = 1)
       onView(

--- a/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt
+++ b/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt
@@ -20,7 +20,6 @@ import androidx.test.espresso.intent.matcher.IntentMatchers.hasExtra
 import androidx.test.espresso.matcher.ViewMatchers.hasDescendant
 import androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA
 import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
-import androidx.test.espresso.matcher.ViewMatchers.isRoot
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.ext.junit.runners.AndroidJUnit4
@@ -50,7 +49,6 @@ import org.oppia.android.app.story.StoryActivity
 import org.oppia.android.app.topic.TopicActivity
 import org.oppia.android.app.topic.TopicTab
 import org.oppia.android.app.utility.EspressoTestsMatchers.withDrawable
-import org.oppia.android.app.utility.OrientationChangeAction.Companion.orientationLandscape
 import org.oppia.android.domain.classify.InteractionsModule
 import org.oppia.android.domain.classify.rules.continueinteraction.ContinueModule
 import org.oppia.android.domain.classify.rules.dragAndDropSortInput.DragDropSortInputModule

